VmAgent.Core.UnitTests/DockerContainerEngineTests.cs

```diff
@@ -24,27 +24,10 @@ public void TestGetImageNameWithTag()
                 ImageDigest = null,
                 Registry = null,
             };
-            Assert.AreEqual("name:tag", DockerContainerEngine.GetImageNameFromContainerImageDetails(imageDetails));
+            Assert.AreEqual("name:tag", DockerContainerEngine.GetImageNameAndTagFromContainerImageDetails(imageDetails));
 
             imageDetails.Registry = "registry";
-            Assert.AreEqual("registry/name:tag", DockerContainerEngine.GetImageNameFromContainerImageDetails(imageDetails));
-        }
-
-        [TestMethod]
-        [TestCategory("BVT")]
-        public void TestGetImageNameWithDigest()
-        {
-            var imageDetails = new ContainerImageDetails
-            {
-                ImageName = "name",
-                ImageTag = "tag",
-                ImageDigest = "digest",
-                Registry = null,
-            };
-            Assert.AreEqual("name@digest", DockerContainerEngine.GetImageNameFromContainerImageDetails(imageDetails));
-
-            imageDetails.Registry = "registry";
-            Assert.AreEqual("registry/name@digest", DockerContainerEngine.GetImageNameFromContainerImageDetails(imageDetails));
+            Assert.AreEqual("registry/name:tag", DockerContainerEngine.GetImageNameAndTagFromContainerImageDetails(imageDetails));
```

Comment:
Would it be worthwhile to also check the individual private methods results equal the new combo method?

Reply:
Nah I'm not too worried about it--if someone changes one of those private methods, this test will fail and we'll figure it out